public/app/features/templating/template_srv.ts

```diff
@@ -200,6 +200,15 @@ export class TemplateSrv implements BaseTemplateSrv {
     if (!target) {
       return false;
     }
+
+    // Scenes compatability
+    if (window.__grafanaSceneContext && window.__grafanaSceneContext.isActive) {
```

Comment:
this means grafana is using scenes?  not sure what the compatibility is.  can we elaborate?

suggestion. might be good to define a couple functions so we aren't using window.__grafanaSceneContext all over the place

// should we export these functions from scenes so anyone using scenes can get the context from the scenes lib?
function sceneContext() {
  return window.__grafanaSceneContext || undefined;
}

function activeSceneContext() {
  return sceneContext()?.isActive;
}